client/app/scripts/components/time-travel-wrapper.js

```diff
@@ -91,27 +135,36 @@ class TimeTravelWrapper extends React.Component {
 }
 
 function mapStateToProps(state, { params }) {
-  const scopeState = state.scope || state;
```

Comment:
This is not important anymore?

Reply:
Nope, `<TimeTravelWrapper />` is now only used in Weave Cloud context, so it's always `state.scope`.